test/index.js

```diff
@@ -5,7 +5,7 @@ const electron = require('electron')
 const chai = require('chai')
 const chaiAsPromised = require('chai-as-promised')
 const path = require('path')
-const fs = require('fs')
+const setup = require('./setup.js')
```

Comment:
FYI the `.js` extension is not required and generally not used in Electron projects. You can just do `require('./setup')`. But if being more explicit feels good here, that's fine with me.